File: src/main/java/PTC/Managers/CommandManager.java
package PTC.Managers;

import PTC.Commands.BroadcastCommand;
import PTC.Commands.FlyCommand;
import PTC.Commands.GamemodeCommand;
import PTC.Commands.SpeedCommand;
import PTC.Core;
import org.bukkit.plugin.Plugin;

public class CommandManager {

    public CommandManager(){
        registerCommands();
    }

    private void registerCommands() {
        Core core = Core.getInstance();
        core.getCommand("bc").setExecutor(new BroadcastCommand());
        core.getCommand("gm").setExecutor(new GamemodeCommand());
        core.getCommand("fly").setExecutor(new FlyCommand());
        core.getCommand("speed").setExecutor(new SpeedCommand());
    }
}

File: src/main/kotlin/PTC/Managers/CommandManager.kt
package PTC.Managers

import PTC.Commands.*
import PTC.Core

class CommandManager {
    init {
        registerCommands()
    }

    private fun registerCommands() {
        val core: Core? = Core.instance
        core?.getCommand("bc")?.executor = BroadcastCommand()
        core?.getCommand("gm")?.executor = GamemodeCommand()
        core?.getCommand("fly")?.executor = FlyCommand()
        core?.getCommand("speed")?.executor = SpeedCommand()
        core?.getCommand("ban")?.executor = BanCommand()
        core?.getCommand("atrcast")?.executor = ATRBroadcast()
        core?.getCommand("pcanima")?.executor = AnimationCommand()
        core?.getCommand("unban")?.executor = UnbanCommand()
    }
}
